test/CommandTest.php

```diff
@@ -98,39 +98,66 @@ public function testCanvasWidthAndHeightNullByDefault()
         $this->assertNull($command->canvasHeight());
     }
 
-    public function testFullTrimMod()
-    {
-        $command = self::command('trimAABBCCx2');
-        $this->assertEquals('AABBCC', $command->trimColor());
-        $this->assertEquals('2', $command->trimFuzz());
+    /**
+     * @param string $strCommand
+     * @param string $color
+     * @param string $fuzz
+     * @dataProvider dpTestOnlyTrimColor
+     */
+    public function testTrim($strCommand, $color, $fuzz)
+    {
+        $command = self::command($strCommand);
+        $this->assertEquals($color, $command->trimColor());
+        $this->assertEquals($fuzz, $command->trimFuzz());
+        $this->assertEquals($strCommand, (string) $command);
+    }
+
+    public function dpTestOnlyTrimColor()
+    {
+        return [
+            [
```

Comment:
Build is failed because of this short array syntax. PHP 5.3 is supported.

Reply:
is 5.3 still actuall

Reply:
4 years passed, yes, now we've dropped 5.3 support.